lib/app/modules/home/views/home_view.dart

```diff
@@ -1424,4 +1453,51 @@ class HomeView extends GetView<HomeController> {
       ),
     );
   }
+
+  Future<bool> showDeleteAlarmConfirmationPopupOnSwipe(
+    BuildContext context,
+  ) async {
+    // Return true if user confirms deletion, false if canceled
+
+    var result = await Get.dialog<bool>(
```

Comment:
This is not what I meant. You'd have to use a dialog consistent with the one you see when you try to navigate back to home from addOrUpdate screen. You can use a `defaultDialog` but follow the same theme (background color, button placements, button style etc.)